app/views/animals/show.html.erb

```diff
@@ -32,7 +32,7 @@
 
       <p>
         <strong>Sex:</strong>
-        <%= @animal.sex %>
+        <%= @animal.sex&.humanize %>
```

Comment:
Is there ever a scenario where there is no sex?

Reply:
not sure. I've added this here because the field is nullable. Should we restrict this @benreyn @CraigJZ  ?

Reply:
Actually yes! In our meeting with stakeholders yesterday, and in the data files they have sent us, there are many cases where the sex of an animal is unknown or not able to be determined until the animal is more mature. 

My preference would be to make the column non-nullable in the database and add a third "unknown" option to the enum.

Reply:
implemented here @haydenrou @benreyn 8631e19ab21ab034bc2c8800347082d18966dac0